src/card/storage_adapters/git_storage_adapter.rb

```diff
@@ -1,22 +1,46 @@
+require_relative "../models.rb"
+require "open-uri"
+
 module Card
   module StorageAdapters
     # Saves the images in the local git repository
     class GitStorageAdapter < Abstract
       # todo move to config
       ASSET_PATH = "assets/core/card/printed"
+      PRINT_MANIFEST_PATH = "assets/core/card/printed/print_manifest.json"
+      GH_URL = "https://raw.githubusercontent.com/elliottomlinson/rpcg"
```

Comment:
I'm prioritizing out own workload right now, but eventually we should do a sweep to get everything into an easy to use config